pennylane/circuit_drawer/circuit_drawer.py

```diff
@@ -46,7 +46,7 @@ class CircuitDrawer:
     Args:
```

Comment:
circuit_drawer and circuit_graph still use the ``indices`` logic, because they are not really a device, and explicitly work with the indices...

Reply:
No, the drawer draws the user wire labels. But it figures out where they are without using the wire map...